library/fluid/fluid_extensions/src/main/java/org/quiltmc/qsl/fluid/impl/FishingBobberEntityExtensions.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2022 QuiltMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.quiltmc.qsl.fluid.impl;
+
+import org.jetbrains.annotations.ApiStatus;
+
+import net.minecraft.fluid.Fluid;
+import net.minecraft.loot.LootTables;
+import net.minecraft.tag.FluidTags;
+import net.minecraft.tag.TagKey;
+import net.minecraft.util.Identifier;
+
+@ApiStatus.Internal
+public interface FishingBobberEntityExtensions {
```

Comment:
Couple things
1. Why is this `@ApiStatus.Internal`? This is a useful API
2. This should have `@InjectedInterface(FishingBobberEntity.class)`